src/app/containers/FeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  /* z-index needs explicitly set as the psammead section label component uses negative z-indices */
+  z-index: 0;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+
+  span {
+    background-color: transparent;
+  }
+
+  span[dir] {
+    background-color: ${C_GHOST};
```

Comment:
Why does the background colour change dependent on the presence of a `dir` attribute?

Reply:
The DOM structure of the Secondary Column Header is:

```html
<h2>
  <span>
    <span dir="...">...</span>
  </span>
</h2>
```

By default the `h2 span` is a gray color, which we do not need, so we make it transparent.

```
span {
    background-color: transparent;
  }
```

But then, there's a gray bar that runs through the text to the right. We need the actual text's background to be white, so the bar is just visible after it. 

That's why we use `span[dir]` to override the bg color for just the span that contains text.

Reply:
Oh okay, so it's a bit of a work around the Secondary Column Header styling, just not that obvious from looking at it. Not sure what the best way to make this clearer is, kinda feel like there should be a prop on `SectionLabel` to make its background transparent but not sure that's worth the effort tbh.